UnitsNet.Tests/CustomCode/VolumeFlowTests.cs

```diff
@@ -96,7 +96,7 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double OilBarrelsPerMinuteInOneCubicMeterPerSecond => 3.77388865E2;
 
-        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 15850.323141489e-3;
+        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 1.5850323141489e1;
```

Comment:
I think this value is correct, but the conversion function is perhaps wrong?

https://ci.appveyor.com/project/angularsen/unitsnet/builds/20278134
Values are not equal within relative tolerance: 0.0010 %
Expected: 15.850323141489
Actual: 15850323.141489
Diff: 99,999,900.0000 %
      Expected: True
      Actual:   False

Reply:
Also a side-question, is KiloUSGallonsPerMinute a widely used unit? I couldn't find any references to it when googling. Only found US gallons per minute, without kilo prefix.

Reply:
I dont think it is widely used, but it is something that we are using in the oil and gas industry

Reply:
FIxed a bug in the conversions :( Sorry for all the bugs! I'm glad you have a good continuous integration pipeline setup, lol.

Reply:
No problem! Yup, the builds sure help :-)